instrumentation/opentelemetry-instrumentation-botocore/examples/bedrock-runtime/zero-code/invoke_agent.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+import boto3
+import json
+import uuid
+import logging
+from botocore.exceptions import ClientError
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+class WeatherAgentDemo:
+    def __init__(self):
+        # Initialize Bedrock Agents Runtime client
+        self.client = boto3.client('bedrock-agent-runtime')
+
+        # Replace these with your actual agent IDs
+        self.agent_id = "TFMZVIWXR7"
```

Comment:
Let's use env vars then

Reply:
changed it on both standalone demo and tests
(so tests now need `export TOX_OVERRIDE=testenv.pass_env=AWS_ACCESS_KEY_ID,AWS_SECRET_ACCESS_KEY,AWS_SESSION_TOKEN,AWS_DEFAULT_REGION,BEDROCK_AGENT_ID,BEDROCK_AGENT_ALIAS_ID`